typing/ctype.ml

```diff
@@ -4131,8 +4131,19 @@ let eqtype_subst type_pairs subst t1 t2 =
   end
 
 let rec eqtype rename type_pairs subst env t1 t2 =
-  if eq_type t1 t2 then () else
-
+  let check_phys_eq t1 t2 =
+    not rename && eq_type t1 t2
+  in
+  (* Checking for physical equality when [rename] is true
```

Comment:
Maybe `physical equality of type representatives` to avoid giving the impression that `eq_type` is `==`?